staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -249,6 +249,13 @@ type ObjectMeta struct {
 	// This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
 	// +optional
 	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
+
+	// LastApplied is a map of workflow-id (user-defined strings) to
+	// last applied configuration serialized as a string. The last
```

Comment:
We can start with strings but I think runtime.Object is going to be better for us. (Imagine seeing quoted json vs. inline yaml when looking at the object printed in yaml.)